docs/install/openshift.md

```diff
@@ -0,0 +1,148 @@
+## Requirements
+
+Before proceeding, please ensure that you have an OpenShift cluster running K8s
+1.19+ (OpenShift 4.7+) and have Helm 3.5+ installed. In addition, you'll need to
+install the OpenShift CLI (`oc`) to authenticate to your cluster and create OpenShift
+resources.
```

Comment:
Is there a reason why customers need this version installed? It could take a customer months to get the proper cluster, so we may want to clarify whether this is a hard requirement or simply what we have tested Coder with.

Reply:
this is just keeping in line with our K8s install docs, which say you need a cluster with version 1.19+ (the OpenShift equivalent is 4.7).